tests/core/search/searchdatatransformer.js

```diff
@@ -39,10 +43,13 @@ describe('tranform vertical search response', () => {
     const data = responseWithNoResults;
     const result = SearchDataTransformer.transformVertical(data);
     const convertedResponse = SearchDataTransformer._reshapeForNoResults(data);
+    const formatters = {};
+
     expect(result).toEqual(
       {
         [StorageKeys.QUERY_ID]: convertedResponse.queryId,
         [StorageKeys.NAVIGATION]: new Navigation(), // Vertical doesn't respond with ordering, so use empty nav.
+        [StorageKeys.DIRECT_ANSWER]: DirectAnswer.fromCore(response.directAnswer, formatters),
```

Comment:
I know this code has already been through PR, but do we need to specify formatters here? I think most of the time people don't specify fieldFormatters